File: algorithm/python/BOJ10951.py
# problem : https://www.acmicpc.net/problem/10951

import sys

answer_list = list()

while True:
    try:
        A, B = map(int, sys.stdin.readline().rstrip().split())
        answer_list.append(A + B)
    except:
        break

for value in answer_list:
    print(value)

File: algorithm/typescript/BOJ10951.ts
// problem : https://www.acmicpc.net/problem/10951

import fs = require("fs");

const input: string[] = fs.readFileSync("/dev/stdin").toString().split("\n");
const result: number[] = [];

for (let i = 0; i < input.length - 1; i++) {
  const inputData: number[] = input[i].split(" ").map(Number);

  const A: number = inputData[0];
  const B: number = inputData[1];

  result.push(A + B);
}

console.log(result.join("\n"));
